### DISCUSSION TITLE
Why can't my simulation converge

### DISCUSSION THREAD

**Discussion by abc-hy** [idaholab/moose#24512] [Q&A Modules: Phase field]
Hi everyone,

My simulation cannot converge. What is the problem with my code? 
Currently, I have a square model, and the initial condition of this square model is c=0.0001, the free energy function is shown as below:
![deltaf_0d4](https://github.com/idaholab/moose/assets/63679626/d356d322-3c27-4868-a784-a02ee287275a)
So, phase 0 will decompose to -1 phase and 1 phase. However, after I run the simulation, it always cannot converge.
This is the error message:
[error_message.txt](https://github.com/idaholab/moose/files/11580746/error_message.txt)
This is my file:
[file.txt](https://github.com/idaholab/moose/files/11580747/file.txt)
How should I change my code?

Thank you,
Best

**Comment by GiudGiud** [comment]
Hello

```
Time Step 2, time = 0.00749663, dt = 0.00300763
 0 Nonlinear |R| = [32m8.079562e-16[39m
    |residual|_2 of individual variables:
                          c: 4.60316e-16
                          w: 6.64005e-16
```

this makes me think you are trying to over-converge things. 
Please turn on automatic_scaling in the Executioner block.
If that does not do it, you ll have to fix the convergence criteria (nl_abs_tol)
